examples/database_silo/main.tf

```diff
@@ -153,8 +153,9 @@ module "postgresDb" {
 resource "transcend_data_silo" "database" {
   type = "database"
 
-  schema_discovery_plugin {
+  disco_class_scan_config {
     enabled                    = true
+    type                       = "SCHEMA_ONLY"
```

Comment:
`schema_discovery_plugin` should now be controlled through a "SCHEMA_ONLY" `disco_class_scan_config`